app/models/shipit/stack.rb

```diff
@@ -251,8 +251,25 @@ def lock_reverted_commits!
       end
     end
 
-    def undeployed_commits
+    def active_commits
+      return [] unless active_task?
+
+      scope =
+        commits
```

Comment:
\nit commits should be on the same line as scope + the first method for readability.
```
scope = commits.reachable
                             .newer_than(active_task.since_commit)
                             .until(active_task.until_commit)